app/views/console/users/user.phtml

```diff
@@ -165,7 +195,7 @@
                                         data-analytics-event="submit"
                                         data-analytics-category="console"
                                         data-analytics-label="Delete User Session"
-                                        data-service="users.deleteSession"
+                                        data-service="users.updateStatus"
```

Comment:
Any reason you changed this action?

Reply:
Shouldn't have been, just of been a left over from when I was experimenting. I'll revert it back